neighbour-model/src/main/java/no/vegvesen/ixn/federation/model/PrivateChannel.java

```diff
@@ -22,19 +21,44 @@ public class PrivateChannel {
     @Column
     private String queueName;
 
+    @Column
+    private String serviceProviderName;
+
+    @OneToOne(cascade = CascadeType.ALL, fetch = FetchType.EAGER, orphanRemoval = true)
+    @JoinColumn(name="end_id", foreignKey = @ForeignKey(name="fk_end_privatechannel"))
+    private PrivateChannelEndpoint endpoint;
+
     public PrivateChannel() {
 
     }
 
```

Comment:
Should ony have 3 constructors. An emtpy, one with peer, sp and status, and one with the three formentioned, as well as a privateChannelEndpoint